tests/Microsoft.Identity.Test.Common/TestConstants.cs

```diff
@@ -124,13 +130,18 @@ public static string Defaultx5cValue
 
         public const string Bearer = "bearer";
 
-        public static readonly IDictionary<string, string> s_extraQueryParams
-            = new Dictionary<string, string>()
+        public static IDictionary<string, string> ExtraQueryParams

```

Comment:
Should this be ExtraQueryParameters? Unless 'Params' is used throughout and for consistency.